stylesheets/components/card.css

```diff
@@ -0,0 +1,131 @@
+@import url(/stylesheets/abstract/variables.css);
+@import url(/stylesheets/abstract/color.css);
+@import url(/Stylesheets/main.css);
+
```

Comment:
Import stylesheets either on Html or Css, don't do both. Also import all sheets in main.css and use that everywhere.

Reply:
okay done